gulpfile-extensions-example.js

```diff
@@ -14,7 +14,9 @@ module.exports = taskMethods => {
     "use strict";
 
     const gulp = require("gulp");
-
+	const ON_DEATH = require("death")({ debug: false });
```

Comment:
Whitespace is off

Reply:
@erik-ciq please get with andy/confirm this comment is resolved.